pkg/controller/windowsmachine/windowsmachine_controller.go

```diff
@@ -73,11 +74,19 @@ func newReconciler(mgr manager.Manager, clusterServiceCIDR string) (reconcile.Re
 		return nil, errors.Wrapf(err, "error creating signer using private key: %v", wkl.PrivateKeyPath)
 	}
 
+	serviceCIDR, err := networkConfig.GetServiceCIDR()
+	if err != nil {
+		return nil, errors.Wrap(err, "error getting service CIDR")
+	}
+
+	vxlanPort := networkConfig.VXLANPort()
```

Comment:
Given there is no error being returned, do we need this variable? I suggest you make the call directly below.